docs/Webhooks/Creating_subscription.rst

```diff
@@ -0,0 +1,55 @@
+.. _creating_subscription:
+
+Creating webhook subscription (Beta version)
+============================================
+
+To create a webhook subscription, send a ``POST`` request to ``/webhooks``.
+The body needs to contain the following JSON data. For example:
+
+  .. code-block:: JSON
+   :linenos:
+    {
+        "description": "Subscription for events",
+        "url": "https://ennpjtc2q1cm.x.pipedream.net",
+        "events": [
+            "OrderCreated"
+        ]
+    }
+
+The response, if successfully will be a JSON object with the following data:
+
+  .. code-block:: JSON
+   :linenos:
+
+    {
+        "success": true
+    }
+
+If you can no longer create a webhook subscription, you will receive a ``500 Internal Server Error``
```

Comment:
Should we change so this gives a more appropriate response code?

Reply:
Oh it does, in case of unauthorized user, it will return 401. just that I have not mentioned it here. And. 500 for other unknown external issues.

Reply:
Yeah, it's a bit weird to explain 500 as you can no longer create a webhook subscription

As for me - it would be nice to say: In case of unexpected exception at Printdeal API side you will receive 500 error. Contact support. [link to contact us page]